deps/deps.go

```diff
@@ -46,6 +47,12 @@ func BinaryWithEnv(name, ver string, binDir string, env map[string]string) Binar
 	return func(msg string, args ...interface{}) error {
 		bin := fmt.Sprintf("%s/%s", binDir, name)
 		InstallDependency(name, ver, binDir)
+		customName := dependencies[name].BinaryName
+		if customName != "" {
+			templated := utils.Interpolate(customName, map[string]string{"os": runtime.GOOS, "platform": runtime.GOARCH})
+			bin := fmt.Sprintf("%s/%s", binDir, templated)
```

Comment:
you just update bin and use a single return - no need to  have 2 returns.
```suggestion
			bin = fmt.Sprintf("%s/%s", binDir, templated)
```